### PR TITLE
updated fix_missing_thumbs.py file

### PR DESCRIPTION
**Pull request by orzut**

### CODE REVIEW

File: src/modules/img_thumb_gen/scripts/fix_missing_thumbs.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 def generate_thumbnails(data, context):
   # Check if the file format is correct
-  if not data['name'].endswith('jpg') or not data['name'].endswith('jpeg'):
+  if not data['name'].endswith(('jpg', 'jpeg', 'JPG', 'JPEG')):
```
Comment (r-vieira) [inline src/modules/img_thumb_gen/scripts/fix_missing_thumbs.py]: can we cover any case combinations? eg: convert the data['name'] to lower case and compare against it. Worry we might see Jpg or Jpeg, etc.

**Review by r-vieira — CHANGES_REQUESTED** [review]
see note about case sensitivity

File: src/modules/img_thumb_gen/scripts/fix_missing_thumbs.py (hunk 1 above)
Comment (orzut) [reply src/modules/img_thumb_gen/scripts/fix_missing_thumbs.py]: pushed the changes in the commit [13c4906](https://github.com/AndersenLab/CAENDR/pull/315/commits/13c49060e1a4de8af314bba437123fa07da95836)

File: src/modules/img_thumb_gen/scripts/fix_missing_thumbs.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 def generate_thumbnails(data, context):
   # Check if the file format is correct
-  if not data['name'].endswith(('jpg', 'jpeg', 'JPG', 'JPEG')):
+  if not data['name'].lower().endswith(('jpg', 'jpeg')):
```
Comment (r-vieira) [inline src/modules/img_thumb_gen/scripts/fix_missing_thumbs.py:26]: great. Sry, should have mentioned to keep the '.' as part of the file extension (".jpg", ".jpeg").

**Review by r-vieira — APPROVED** [review]
approved

**Pull request merged**